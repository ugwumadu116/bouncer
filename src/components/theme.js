import * as colors from "./color/color";
import * as fontsize from "./text/fontsize";
import * as fontweight from "./text/fontweight";
import * as lineheight from "./text/lineheight";
import * as letterspacing from "./text/letterspacing";
import * as btnSizes from "./button/button";
import * as btnState from "./button/btnState";
import * as inputsizes from './form/input/inputsizes'
import * as inputbuttonSize from './form/inputButton/inputbuttonsizes'
import * as selectSize from './form/select/selectSizes'


const theme = {
  colors,
  fontsize,
  fontweight,
  lineheight,
  letterspacing,
  btnSizes,
  btnState,
  inputsizes,
  inputbuttonSize,
  selectSize
};
export default theme;
