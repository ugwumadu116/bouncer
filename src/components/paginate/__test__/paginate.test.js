import React from "react";
import { render, fireEvent } from "@testing-library/react";
import theme from "../../theme";
import { ThemeProvider } from "styled-components";
import Pagination from "..";
import '@testing-library/jest-dom/extend-expect'


test('test background color of active state', () => {
    const { getByText } = render( 
    <ThemeProvider theme={theme}>
        <Pagination />
    </ThemeProvider>
    )
    const bg = getByText('1')
    expect(bg.style.backgroundColor).toBe('black')
    expect(bg.style.color).toBe('white')
    expect(bg.style.backgroundColor).toMatchSnapshot();
})
