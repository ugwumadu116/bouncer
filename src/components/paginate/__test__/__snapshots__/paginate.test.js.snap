// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test background color of active state 1`] = `"black"`;
